ludwig/hyperopt/execution.py

```diff
@@ -806,6 +810,8 @@ def _register(name, trainable):
         # https://docs.ray.io/en/latest/tune/tutorials/tune-stopping.html
         should_resume = "AUTO" if resume is None else resume
 
+        raise_on_failed_trial = kwargs.get("raise_on_failed_trial", True)
```

Comment:
nit: Why not just add this as an optional argument to the function? That was, you can also add documentation for it in the docstring that explains why it's needed.

Reply:
This is an artifact of an earlier build I'm going to remove it now.

Reply:
@jeffkinnison Was this supposed to be updated based on your response to Shreya's comment? Seeing the same thing as before - let me know!

Reply:
Removed in [a073cc0](https://github.com/ludwig-ai/ludwig/pull/2640/commits/a073cc00aec3e3915c4cdd143aabbee7a247c816)